Assets/MixedRealityToolkit/_Core/Utilities/Scripts/HoverLight.cs

```diff
@@ -58,18 +59,16 @@ private void OnDisable()
             UpdateHoverLight();
         }
 
-#if UNITY_EDITOR
         private void Update()
         {
-            if (Application.isPlaying)
+            if (Application.isPlaying || Application.isEditor)
```

Comment:
should this be && instead of || ? The original code got executed when it was in the editor, this looks like it will occur when the app is running on device as well.

Reply:
Same applies here as well.